src/test/suite/_extension.test.ts

```diff
@@ -45,7 +45,7 @@ suite('Extension Test Suite', () => {
 					}
 				}
 			}
-			expect(match).to.be.true;
+			expect(match, `Tutorial ${tutorialName} not found in registry. It contains ${existingRegistry}`).to.be.true;
```

Comment:
why did u rename the file?

Reply:
ah forgot to submit the comment. Good question.  It is to be sure that it is played first. This test is expecting that the registry was not cleared whcih is something that some other tests are doing. Using the underscores ensures that it is played first.

Reply:
sounds scary....but at least it seems to work :D